src/function/special/riemannZeta.js

```diff
@@ -0,0 +1,87 @@
+import { factory } from '../../utils/factory.js'
+
+const name = 'riemannZeta'
```

Comment:
In Matlab and Mathematica for example you can use `zeta`. I see in your unit tests you prefer this short name `zeta` too. Is it an idea to just name the function `zeta` instead of `riemannZeta`?

Reply:
I named it riemannZeta since there are a bunch of zeta functions out there that I didn't want to confuse with such as the Hurwitz Zeta function. Since the Riemann zeta is the most well known of the zeta functions I think it's okay to rename it to zeta and if subsequent zeta functions are implement to name them hurwitzZeta etc

Reply:
Still planning on renaming the function and files `zeta`?